freshteam/iparams-json/app/app.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready( function() {
+    app.initialized()
+        .then(function(_client) {
+          var client = _client;
+          client.events.on('app.activated',
+            function() {
+                client.data.get('jobPosting')
+                    .then(function(data) {
+                        $('#apptext').text("Job Title: " + data.title);
+                    })
+                    .catch(function(e) {
+                        console.log('Exception - ', e);
+                    });
+        });
+    });
+});
```

Comment:
App looks great, can you add in `client.iparams.get()` retrieving iparam values and displaying them on the app?

Reply:
Sure, if we include client.iparams.get() it gives more clarity. we will let you know once changes have been done

Reply:
@zachjonesnoel Included feature of retrieving iparams values in the app. Please review & let us know the update

Reply:
@zachjonesnoel Did you get a chance to review the changes?